admin/Templates/article.html.php

```diff
@@ -0,0 +1,44 @@
+</div>
+<div class="parallax-container">
+    <div class="parallax">
+        <img src="../public/assets/images/posts/<?= $article->getImage() ?>" alt="<?= $article->getTitle() ?>"/>
```

Comment:
```suggestion
        <img src="/assets/images/posts/<?= $article->getImage() ?>" alt="<?= $article->getTitle() ?>"/>
```

Ca ne marche pas comme ca?